src/SDKs/_metadata/keyvault_resource-manager.txt

```diff
@@ -1,10 +1,36 @@
-2018-03-22 23:02:34 UTC
+﻿Installing AutoRest version: latest
 
+> autorest@2.0.4280 preinstall C:\Program Files\nodejs\node_modules\.staging\autorest-96e45455
+> node ./preinstall-check
```

Comment:
nit: please pull down latest changes from upstream and run `msbuild build.proj` at the repo and regenerate the code. The metadata here should be much less.
Not a blocker